### ISSUE TITLE
cryptic error when split_and_predict generates no predictions

### ISSUE THREAD

**Issue by sammlapp** [kitzeslab/opensoundscape#417]
If the files are bad or there's any sort of bug that leads to failed preprocessing (wrong file paths, etc), this cryptic error is given. It should instead handle this scenario and perhaps print a warning that no predictions were generated. 

```
Warning: Given Audio object with duration of `0.506375` seconds and `clip_duration=5.0` but  `final_clip=None` produces no clips. Returning empty list.
  warnings.warn(
Traceback (most recent call last):
  File "predict.py", line 43, in <module>
    score_df, _, unsafe_samples = model.split_and_predict(
  File "/home/kitzeslab/miniconda3/envs/opso060/lib/python3.8/site-packages/opensoundscape/torch/models/cnn.py", line 897, in split_and_predict
    total_scores = np.concatenate(total_scores, axis=0)
  File "<__array_function__ internals>", line 5, in concatenate
ValueError: need at least one array to concatenate
```

**Comment by sammlapp** [comment]
I think this may happen for any prediction, not just split_and_predict

**Comment by sammlapp** [comment]
resolved in #423
